apps/examples/timer/timer_main.c

```diff
@@ -110,7 +110,7 @@ static pthread_addr_t timer_thread(pthread_addr_t arg)
 	int intval = pargs->intval;
 	int fd = pargs->fd;
 #ifdef CONFIG_EXAMPLES_TIMER_FRT_MEASUREMENT
-	int frt_fd;
+	int frt_fd = -1;
```

Comment:
Is there any difference between 0 and -1?
How about 0 instead of -1? Because -1 should be saved at somewhere, but 0 isn't.